### DISCUSSION TITLE
My proposal for a new way to play the game.

### DISCUSSION THREAD

**Discussion by Griorite** [CleverRaven/Cataclysm-DDA#48272] [Ideas]
This is my first time ever posting on GitHub so please excuse me if this post is to long or uninformative. I have been playing this game for over two years now and over that time, I have come up with a new set of content that I think would benefit the game.

My idea is to implement a train system into the world. I'm talking tracks that covered the map in believable ways, new points of interest such as train stations and train yards, and new vehicle tiles specifically made for trains. I adore the thought of finding a train, customizing it to my liking, and letting the tracks take me to wherever they go. Since these trains wouldn't have to be steered, you could craft, read, and even sleep while the train continues to move, without the worry of crashing. The trains could be made of a new type of frame that is more durable than even heavy frames but the tradeoff would be these new frames could be incredibly heavy, meaning they are best suited for use on tracks. A new type of heavy plating could also be introduced that is not only durable, but allows this. What if the floors of the train cars could support structures on them. An example of which is you could build beds and furniture without them being directly tied into your vehicle. The trains could have a system where you could attach and build train cars to one another to allow your train to be as long or short as you want. The trains could have a derailment system where going two fast around turns or hitting objects (such as trees or buildings, not zombies) could cause your train to come off the tracks and become stuck. Finally, I don't know if you would need special knowledge to operate the trains like with helicopters. That would be up to you guys.

Now, I am not a developer or a coder, I'm just an average player, so these things I'm saying may be impossible to implement in the current state of the game. If so, I totally understand, as I can't even contribute to the game myself. But I just hope you guys see this post and at least consider if something you would like I your game. Thank you for reading this and I look forwards to keep playing the game.

**Comment by Raikiri** [comment]
There already are trains (in subways). Even though they're kinda rare, they already do most of what you're describing.

**Reply by actual-nh** [reply]
Most of the above appears to be targeted at above-ground exploration.

**Reply by actual-nh** [reply]
One other thought - if you're wanting to represent New England, there really should be quite a lot of above-ground trains, including both freight and passenger.

**Comment by ThePirate42** [comment]
I think it would be better if the title of the discussion specified that this is a proposal about trains. As it is, it's a little generic.
